### PR TITLE
Add clean_up bin script.

### PR DESCRIPTION
**Pull request by pmeijer**
Added utility method to get gmeConfig with requirejs paths resolved (this script does not use it yet and works with 1.5.1).

`npm run clean_up -- -h`

### CODE REVIEW

File: src/bin/clean_up.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        .version('1.6.0')
+        .option('-d, --del [boolean]', 'If true will do the deletion [false].', false)
+        .option('-l, --list [boolean]', 'If true will list all the projects this user has access to [false].', false)
+        .option('-t, --daysAgo [number]', 'Minimum age (last viewed) of a project to delete [10].', 10)
+        .option('-c, --commits [number]', 'Maximum number of commits of a project to delete [1].', 1)
+        .option('-b, --branches [number]', 'Maximum number of branches of a project to delete [1].', 1)
+        .option('-r, --regex [string]', 'Project names must match the regexp [.*].', '.*')
+        .option('-u, --username [string]', 'the user that will remove the projects [guest account]')
+        .on('--help', function () {
+            console.log('  Examples:');
```
Comment (lattmann) [inline src/bin/clean_up.js]: For the examples, I suggest to use the long version of the commands, for instance:

    node clean_up.js --list

**Comment by pmeijer** [comment]
Fixed PR comments

**Comment by lattmann** [comment]
Have you checked it in?

**Comment by pmeijer** [comment]
https://github.com/webgme/webgme/commit/6e85ef4f07d8885c80b932e7730c8db0e8358814

**Comment by lattmann** [comment]
Looks good, thanks.

**Pull request merged**